data/transport missions.txt

```diff
@@ -1679,6 +1676,23 @@ mission "Shady passenger transport 3 - double cross"
 
 
 
+mission "Shady passenger transport 3 - double cross big payment"
+	name "Shady passenger transport big payment"
+	landing
+	invisible
+
+	to offer
+		has "bribed mercenaries"
+
+	on offer
```

Comment:
Why not just
```
on offer
	payment -8000000
	conversation
		`You receive a message informing you of an unusually large transaction: 8 million credits. The gang of cutthroats must have withdrawn the money you bought them off with. You are now free to continue your career - poorer, but hopefully wiser.`
			decline
```
?

Reply:
Oh, that's clever. I was working around the fact that `dialog` gets Accept and Decline buttons in an `on offer` tag, but a `conversation` would do it too.